test/Microsoft.Azure.SignalR.Tests/AddAzureSignalRFacts.cs

```diff
@@ -85,8 +89,41 @@ public void AddAzureReadsConnectionStringFirst()
 
             var options = serviceProvider.GetRequiredService<IOptions<ServiceOptions>>().Value;
 
-            Assert.Equal("myconnectionstring", options.ConnectionString);
-            Assert.Equal("myconnectionstring", capturedConnectionString);
+            Assert.Equal(DefaultValue, options.ConnectionString);
+            Assert.Equal(DefaultValue, capturedConnectionString);
+        }
+
+        [Theory]
+        [InlineData(CustomValue, null, null, CustomValue)]
+        [InlineData(CustomValue, DefaultValue, SecondaryValue, CustomValue)]
+        [InlineData(null, DefaultValue, SecondaryValue, DefaultValue)]
+        [InlineData(null, null, SecondaryValue, SecondaryValue)]
+        public void AddAzureSignalRLoadConnectionStringOrder(string customValue, string defaultValue,
+            string secondaryValue, string expected)
+        {
+            var services = new ServiceCollection();
+            var config = new ConfigurationBuilder()
+                            .AddInMemoryCollection(new Dictionary<string, string>
+                            {
+                                {"Azure:SignalR:ConnectionString", defaultValue},
+                                {"ConnectionStrings:Azure:SignalR:ConnectionString", secondaryValue}
+                            })
+                            .Build();
+            var serviceProvider = services.AddSignalR()
```

Comment:
nit: This formatting looks a bit indented, can we reduce it a little

Reply:
Sure.